llm/llama.go

```diff
@@ -63,8 +63,8 @@ type ModelRunner struct {
 	Accelerated bool
 }
 
-func chooseRunners(workDir, runnerType string) []ModelRunner {
-	buildPath := path.Join("llama.cpp", runnerType, "build")
+func chooseRunners(workDir string) []ModelRunner {
```

Comment:
As a follow up PR we should rework this to just choose the single runner and instead edit the number of GPU layers based on the arch